app/templates/services/_edit-status.html

```diff
@@ -1,18 +1,22 @@
-{% if service_data.status != 'disabled' %}
-  <form action="{{ url_for('.update_service_status', service_id=service_id ) }}" method="POST">
-    <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
-      {% include "toolkit/forms/selection-buttons.html" %}
-    <button type="submit" class="button-save">Save and return</button>
-  </form>
-{% endif %}
-{% if service_data.status == 'disabled' %}
-  <div class="question">
-    <h2 class="question-heading service-status-removed">
-      This service has been removed
-    </h2>
-    <p class="hint">
-      If you don't know why this service has been removed, contact
+{% if service_data.status == 'published' %}
```

Comment:
This file could probably be renamed from `_edit-status.html` to something along the lines of `_remove-service.html`